start-up/moderation-tutorial.md

```diff
@@ -1,3 +1,69 @@
 # Moderation Tutorial
 
-TODO
\ No newline at end of file
+## Intro
+
+This guide will give you the basic information needed to moderate your Discord server with MoonlightBot. It includes a list of commands and how they work, complete with their arguments, infractions, and log information to help you better understand how to use them
+
+## Arguments
+
+Arguments are options you may use while executing moderation commands. You can find a more in-depth guide about arguments in the [arguments page](/start-up/arguments.md)
```

Comment:
I was thinking of having this introduce them as the most commonly seen types of options in mod commands

Reply:
What do you mean by this, in order of most commonly used?

Reply:
No, mention these options are commonly seen in mod commands, to show the purpose of that section